packages/shared/src/components/fields/MarkdownInput/CommentMarkdownInput.tsx

```diff
@@ -4,8 +4,10 @@ import React, {
   FormHTMLAttributes,
   ReactElement,
   useEffect,
+  useLayoutEffect,
   useRef,
 } from 'react';
+import { FocusOn } from 'react-focus-on';
```

Comment:
Is this a new package? Seems like not used and not found in our `package.json`.

Reply:
that's just something I was testing locally, removed 👍